pkg/bot/socketslack.go

```diff
@@ -519,6 +538,21 @@ func resolveBlockActionCommand(act slack.BlockAction) string {
 		//   @Botkube kcc --verbs get
 		//   @Botkube kcc --resource-type
 		command = fmt.Sprintf("%s %s", act.ActionID, act.SelectedOption.Value)
+	case "plain_text_input":
+		// The main motivation is to extract original command this input belongs to.
+		// a. If the command is located in section blocks, we extract it from section block text
+		// b. If the original command is located inside an input block, we use blockID since the original
+		// command is propagated via that field.
+		blockType := callback.Message.Msg.Blocks.BlockSet[0].BlockType()
+		originalCmd := ""
+		if blockType == slack.MBTSection {
+			originalCmd = callback.Message.Msg.Blocks.BlockSet[0].(*slack.SectionBlock).Text.Text
+		} else if blockType == slack.MBTInput {
+			// blockID contains previously executed command.
+			originalCmd = callback.Message.Msg.Blocks.BlockSet[0].(*slack.InputBlock).BlockID
+		}
+		cmd := strings.Split(originalCmd, "`")[1]
+		command = fmt.Sprintf("%s %s --filter %s", b.BotName(), cmd, command)
```

Comment:
```suggestion
		command = fmt.Sprintf("%s %s --filter=%q", b.BotName(), cmd, command)
```
It will be just less error-prone and more readable if we will do `--filter="  foo bar"` instead of `--filter=  foo bar`
![Screen Shot 2022-10-12 at 18 12 21](https://user-images.githubusercontent.com/17568639/195394540-b91bd182-d130-4107-b409-e0d2d04fbc9b.png)